tests/recipes/test_eleuther_eval.py

```diff
@@ -122,3 +122,40 @@ def test_eval_recipe_errors_without_lm_eval(self, caplog, monkeypatch, tmpdir):
 
         err_log = caplog.messages[-1]
         assert "Recipe requires EleutherAI Eval Harness v0.4" in err_log
+
+    @pytest.mark.integration_test
+    def test_eval_recipe_errors_with_generate_until_and_mc_tasks(
```

Comment:
more tests good, yes?

Reply:
I though @SalmanMohammadi added a test very similar to this already? Also test isn't added?

Reply:
I've decided to be a bad person and add it in a follow-up.

Reply:
I can add this back in when I fix this.